src/DelegationRegistry.sol

```diff
@@ -3,76 +3,76 @@
 pragma solidity ^0.8.16;
 
 import {EnumerableSet} from "openzeppelin-contracts/contracts/utils/structs/EnumerableSet.sol";
+import {ERC165} from "openzeppelin-contracts/contracts/utils/introspection/ERC165.sol";
+import {IDelegationRegistry} from "./IDelegationRegistry.sol";
 
 /** 
 * @title An immutable registry contract to be deployed as a standalone primitive
-* @author foobar
+* @author foobar, wwhchung
```

Comment:
Somewhat of a nit that isn't specific at all to your PR @wwhchung but think it may be a better format for an effort like this to recognize all the folks contributing to this conversation via a "contributors" comment block that includes recognizing folks in the Telegram, etc. who may not be making code-updates.

Just wanted to note as an idea in interest of inclusivity :)

Reply:
Ha, fair, although I don't know what the pattern is here.  Up to @0xfoobar to make a call. :)

Reply:
Yep yep–agreed, I have no strong opinion and not a reflection of your PR at all, just something I noticed and figured worth starting a conversation around.

Reply:
Filed an issue to follow up on this item separately: https://github.com/0xfoobar/nft-delegation/issues/15

Resolving this convo to make it clear this is not intended to be a blocker for this PR.

Reply:
jk... I can't resolve comments it seems :kek: